src/streamlink/plugins/wwenetwork.py

```diff
@@ -37,109 +38,133 @@
+                    ),
+                    validate.all(
+                        {"authorisationToken": str},
+                        validate.get("authorisationToken"),
+                        validate.transform(lambda data: (True, data)),
+                    ),
                 ),
-            )
+            ),
+        )
+
+        if not success:
+            log.error(data)
+            return
+
         return data
 
-    def login(self, email, password):
-        log.debug("Attempting login as {0}".format(email))
-        # sets some required cookies to login
-        data = self.request(
-            "POST",
-            self.login_url,
-            data=json.dumps({"id": email, "secret": password}),
-            headers={"Content-Type": "application/json"},
+    def _get_streams_content(self, content_type, content_id, token):
+        success, data = self.session.http.get(
+            self._API_URLS.get(content_type).format(content_id),
+            acceptable_status=(200, 401, 404),
+            params={"includePlaybackDetails": "URL"},
+            headers={"Authorization": f"Bearer {token}", **self._API_HEADERS},
+            schema=validate.Schema(
+                validate.parse_json(),
+                validate.any(
+                    validate.all(
+                        {"messages": validate.list(str)},
+                        validate.get(("messages", 0)),
+                        validate.transform(lambda message: (False, message)),
+                    ),
+                    validate.all(
+                        {
+                            "accessLevel": str,
+                            "id": int,
+                            "title": str,
+                            validate.optional("playerUrlCallback"): validate.any(None, validate.url()),
+                        },
+                        validate.union_get(
+                            "accessLevel",
+                            "playerUrlCallback",
+                            "id",
+                            "title",
+                        ),
+                        validate.transform(lambda data: (True, data)),
+                    ),
+                ),
+            ),
+        )
+
+        if not success:
+            log.error(data)
+            return
+
+        access, playback_url, self.id, self.title = data
+
+        if access != "GRANTED":
+            log.error("Paid subscription required for this video")
+            return
+
+        playback_schema = validate.Schema(
+            validate.all(
+                {
+                    "url": validate.url(),
+                    validate.optional("subtitles"): validate.none_or_all(
+                        [{"language": str, "format": str, "url": validate.url()}],
+                        validate.filter(lambda s: s["format"] == "srt"),
+                    ),
+                },
+                validate.union_get("url", "subtitles"),
+            ),
```

Comment:
`validate.Schema` already is a subclass of `validate.all`, so the inner `validate.all` can be removed